decredPoliteia/decredPoliteia_outline.md

```diff
@@ -7,9 +7,9 @@
 **main idea**: Pi is the _ideal_ infrastructure for the self-governance of a cryptocurrency.
 
 - Why?
-  - Solves technical challenges associated with self-governance of a cryptocurrency
+  - Solves technical and social challenges associated with self-governance of a cryptocurrency
 - what is self-governance? _(or what is needed to be successful?)_
```

Comment:
I guess this is meant more as a philosophical question. Should pertain not just to decred specifically but governance of a cryptocurrency as a whole. maybe more "what would the perfect governance look like?" or maybe "what problems should governance solve"

Reply:
Governance is making the rules. Decred's rules change when stakeholders vote to approve a change to those rules, but there's more to governance than that. Governance means having the power to set a course for the project and fund development in that direction, without that's it's just a question of choosing a developer group to follow. Politeia is about this aspect of Decred's governance.

Reply:
Right. so lets focus on that aspect. Just really simplify it and say:
1.  for 'good' or 'open' (<-- not sure about best word here) governance of a cryptocurrency you need a way for users to set the course for the project. 
2. That presents these technical challenges. 
3. Here's how Decred developers solved them in Pi.
4. Heres how Decred utilizes pi.

It would be nice to describe all aspects of decred's governance but thats not really in the scope of this video and it would end up being too long. 

The goal as I see it is to use Pi to spark interest in Decred. The governance debate is pretty hot right now so id like to presnt Pi as a general solution for governance infrastructure for all projects. Then, if people want to know more specifics about pi or decred they can read the docs or watch other videos with more focus on details.

Reply:
3 and 4 could possibly be combined